src/index.js

```diff
@@ -95,7 +95,9 @@ class TwitterLogin extends Component {
         'Content-Type': 'application/json'
       }
     }).then(response => {
-      this.props.onSuccess(response);
+      return response.json();
```

Comment:
I would not accept this change. By returning just body of response as json object we will remove any data that response can contain in headers. Because of that I would prefer to return whole response and let user decide if they doesn't need anything else but body. Could you revert that?